python/paddle/incubate/passes/pir.py

```diff
@@ -470,7 +470,7 @@ def RegisterPass(function=None, input_specs={}):
         .. code-block:: python
 
             >>> import paddle
-            >>> from paddle.base.ir import RegisterPass
+            >>> from paddle.base.pir import RegisterPass
```

Comment:
```suggestion
            >>> from paddle.incubate.passes.ir import RegisterPass
```

这里看起来是原来写的就有问题，辛苦改一下～